css/types/-moz-image-rect.json

```diff
@@ -0,0 +1,69 @@
+{
+  "css": {
+    "types": {
+      "-moz-image-rect": {
+        "__compat": {
+          "description": "<code>-moz-image-rect()</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-moz-image-rect",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "2"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "qq_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false,
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=32177'>WebKit bug 32177</a>"
+            },
+            "safari_ios": {
+              "version_added": false,
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=32177'>WebKit bug 32177</a>"
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "uc_android": {
+              "version_added": false
+            },
+            "uc_chinese_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

Comment:
I think this is experimental. The goal was to get it standardized but this didn't get done due to lack of other browser vendor interest.